sdks/python/apache_beam/coders/coders.py

```diff
@@ -1039,11 +1039,16 @@ def __hash__(self):
 
   @classmethod
   def from_type_hint(cls, typehint, unused_registry):
-    if issubclass(typehint, proto_utils.message_types):
+    # The typehint must be a subclass of google.protobuf.message.Message.
+    # ProtoCoder cannot work with message.Message itself, as required APIs are
+    # not implemented in the base class. If this occurs, an error is raised
```

Comment:
Perhaps clarify the issue a bit more, e.g.

ProtoCoder cannot work with message.Message itself, as deserialization of a serialized proto requires knowledge of the desired concrete proto subclass which is not stored in the encoded bytes themselves.

Reply:
Done.